packages/shorebird_cli/lib/src/third_party/flutter_tools/lib/src/base/io.dart

```diff
@@ -71,6 +70,12 @@ ExitFunction get exit {
 bool _inUnitTest() {
   return Zone.current[#test.declarer] != null;
 }
+
+/// Thrown when [exit] is called unexpectedly in a test.
+class UnexpectExitCalled implements Exception {
+  /// Creates a new [UnexpectExitCalled] exception.
+  const UnexpectExitCalled();
+}
```

Comment:
I'm not sure this (or `setExitFunctionForTests`) is necessary now that we aren't using `exit`

Reply:
My goal with this, is that we add this call to the tests, this will make sure we (or other devs) don't unintentionally call `exit`.

I don't mind removing if you think I am being too overcautious here

Reply:
The tests just exit early (and fail) if exit is called, so I think it would probably be fine to remove.

Reply:
hum, yeah, you are actually right! Gonna remove this whole file.